src/pages/[platform]/build-ui/formbuilder/special-inputs/index.mdx

```diff
@@ -33,106 +33,80 @@ export function getStaticProps(context) {
-3. Click the blue plus sign to add a new field to your form
-4. Select a `StorageField` input element
-
-![screenshot of a storage field being added to an unconnected form](/images/console/formbuilder/storage3.png)
-
-With all unconnected forms, the form submission must be configured manually. Use the `onSubmit` hook and the `fields` object to [handle submission in code](/gen1/[platform]/build-ui/formbuilder/lifecycle/#handle-form-data-submissions---onsubmit).
-
-### Configuring Storage Manager
-
-Storage Manager has several unique configurations that can be applied in the right-hand toolbar.
+### Adding it to your form
+
+To use the StorageManager component with an autogenerated form you will first need a data model that has an attribute that is either a string or an array of strings (`a.string().array()` in **amplify/data/resource.ts**). Then make sure to run `npx amplify generate forms` after you update your data model. 
+
+Then go into the generated form JSX file you want to use the StorageManager, for example: **ui-components/TodoCreateForm.jsx**. If your attribute is an array of strings, look for an `<ArrayField>` with `items={images}` (if your attribute name is "images"). Remove that entire component and replace it with the StorageManager component like this:
+
+```jsx title="ui-components/TodoCreateForm.jsx"
+// imports
+import { StorageManager } from "@aws-amplify/ui-react-storage";
+// import the processFile helper function which will create unique filenames based on the file contents
+import { processFile } from "./utils";
+
+//...
+<StorageManager
+  accessLevel="public"
+  maxFileCount={10}
+  acceptedFileTypes={['image/*']}
+  processFile={processFile}
+  onUploadSuccess={({key}) => {
+    // assuming you have an attribute called 'images' on your data model that is an array of strings
+    setImages(prevImages => [...prevImages, key])
+  }}
+  onFileRemove={({key}) => {
+    setImages(prevImages => prevImages.filter(img => img !== key))
+  }}
+/>
+```
 
-- **File level access**: determines which users can access the uploaded file. Storage Manager will default to Private.
-  - _Private_: only accessible to the uploader
-  - _Protected_: accessible to any authenticated user
-  - _Public_: accessible to anyone
-- **Accepted file types**: limits which file types can be uploaded
-- **Advanced settings**
-  - **Maximum number of files**: max number of files that can be added to the form before submission. If your bound data model field is not an array, this will always be 1
-  - **Maximum file size**: max size for each file uploaded
-  - **Is pausable**: allows the user to pause and resume the upload
-  - **Preview image**: if the uploaded file is an image, a preview of the image will be rendered in your form
+If you want your data model to have only 1 image and not an array of images look for the `<TextField>` component with `value={image}` and replace it with the StorageManager component like this:
```

Comment:
```suggestion

If you want your data model to have only one image instead of an array of images, look for the `<TextField>` component with `value={image}` and replace it with the StorageManager component like this:

```